ci/travis-after-success.sh

```diff
@@ -1,6 +1,19 @@
 #!/bin/sh
 set -xe
 
+# Submit code coverage report
+mkdir coverage
+cp .codecov.yml coverage/
+cp ci/codecov.sh coverage/
+find . -maxdepth 1 -type f -name "*.lst" -exec mv '{}' 'coverage/' \;
+CODECOVCMD="codecov.sh -y .codecov.yml -R ."
+docker run -ti --rm -v $PWD/coverage:/docker -w /docker \
+    -e CI \
+    -e TRAVIS -e TRAVIS_BRANCH -e TRAVIS_COMMIT -e TRAVIS_JOB_NUMBER \
+    -e TRAVIS_PULL_REQUEST -e TRAVIS_JOB_ID -e TRAVIS_REPO_SLUG -e TRAVIS_TAG \
+    -e TRAVIS_OS_NAME \
```

Comment:
Not a huge deal, but I think it would be easier to just pass these as [arguments to the script](https://github.com/codecov/codecov-bash/blob/d82a80bc465343ed367fa0788aff312ce5582a61/codecov#L149-L156), the less the script discovers by itself, the better IMHO.